source/Octopus.Cli/Commands/Deployment/DeploymentCommandBase.cs

```diff
@@ -41,18 +41,17 @@ protected DeploymentCommandBase(IOctopusAsyncRepositoryFactory repositoryFactory
             options.Add<bool>("guidedFailure=", "[Optional] Whether to use guided failure mode. (True or False. If not specified, will use default setting from environment)", v => UseGuidedFailure = v);
-            options.Add<string>("specificMachines=", "[Optional] A comma-separated list of machine names to target in the deployed environment. If not specified all machines in the environment will be considered.", v => SpecificMachineNames.AddRange(v.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(m => m.Trim())));
-            options.Add<string>("excludeMachines=", "[Optional] A comma-separated list of machine names to exclude in the deployed environment. If not specified all machines in the environment will be considered.", v => ExcludedMachineNames.AddRange(v.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(m => m.Trim())));
+            options.Add<string>("specificMachines=", "[Optional] A comma-separated list of machine names to target in the deployed environment. If not specified all machines in the environment will be considered.", v => SpecificMachineNames.AddRange(v.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(m => m.Trim())), allowsMultiple: true);
+            options.Add<string>("excludeMachines=", "[Optional] A comma-separated list of machine names to exclude in the deployed environment. If not specified all machines in the environment will be considered.", v => ExcludedMachineNames.AddRange(v.Split(new[] { ',' }, StringSplitOptions.RemoveEmptyEntries).Select(m => m.Trim())), allowsMultiple: true);
             options.Add<bool>("force", "[Optional] If a project is configured to skip packages with already-installed versions, override this setting to force re-deployment (flag, default false).", v => ForcePackageRedeployment = true);
-            options.Add<string>("skip=", "[Optional] Skip a step by name", v => SkipStepNames.Add(v));
+            options.Add<string>("skip=", "[Optional] Skip a step by name", v => SkipStepNames.Add(v), allowsMultiple: true);
             options.Add<bool>("noRawLog", "[Optional] Don't print the raw log of failed tasks", v => noRawLog = true);
             options.Add<string>("rawLogFile=", "[Optional] Redirect the raw log of failed tasks to a file", v => rawLogFile = v);
-            options.Add<string>("v|variable=", "[Optional] Values for any prompted variables in the format Label:Value. For JSON values, embedded quotation marks should be escaped with a backslash.", ParseVariable);
+            options.Add<string>("variable=|v=", "[Optional] Values for any prompted variables in the format Label:Value. For JSON values, embedded quotation marks should be escaped with a backslash.", ParseVariable, allowsMultiple: true);
```

Comment:
Swapped this round to prioritise `--variable` over `-v`.
Also, it was specified wrong previously - it should have been `v=`. Not sure how it worked, but it did...